debian/ubuntu-advantage-tools.postinst

```diff
@@ -278,6 +278,20 @@ remove_old_systemd_units() {
     fi
 }
 
+move_user_config_file() {
+    # User might already have a config-file in /var/lib/ubuntu-advantage
+    # We need to move it to the new location
+    source_file="/var/lib/ubuntu-advantage/user-config.json"
+    destination_dir="/var/lib/ubuntu-advantage/private"
+
+    # Check if the source file exists
+    if [ -f "$source_file" ]; then
+        mkdir -p "$destination_dir"
+        # Move the user-config.json file to the private directory
+        mv "$source_file" "$destination_dir/user-config.json"
+    fi
+}
+
```

Comment:
No need for this to be a separate function. We should have one function to handle the complete migration of the old user-config.json to the new split user-config.json